starters/cms/src/data.ts

```diff
@@ -141,14 +136,21 @@ export async function syncCollection(
         })
     }
 
-    await collection.setFields(sanitizedFields)
     await collection.removeItems(Array.from(unsyncedItems))
     await collection.addItems(items)
 
     await collection.setPluginData(PLUGIN_KEYS.DATA_SOURCE_ID, dataSource.id)
     await collection.setPluginData(PLUGIN_KEYS.SLUG_FIELD_ID, slugField.id)
 }
 
+export const syncMethods = [
+    "ManagedCollection.getFields",
+    "ManagedCollection.getItemIds",
+    "ManagedCollection.removeItems",
+    "ManagedCollection.addItems",
+    "ManagedCollection.setPluginData",
+] as const
```

Comment:
Can plugin authors get help from the library types in managing this?

Reply:
Had this thought briefly, but think `as const` is fine, as ultimately this gets fed into `isAllowedTo`, which will not let you provide invalid methods. This is why I didn't export `NamespacedMethod` from `framer-plugin`.

Thoughts?

Reply:
I think the DUX is pretty bad if you get a type error somewhere completely different from where you are typing